conf/deployment/vsphere/ipi_1az_rhcos_lowerreq_vmfs_3m_3w.yaml

```diff
@@ -0,0 +1,13 @@
+---
```

Comment:
@vavuthu about VMFS, this is basically rely on which DC you run with, right? If I will run on DC ECO it will be VSAN, right?

Reply:
yes. It will rely on whichDC we run, DC-ECO is vSAN

Reply:
@clacroix12 then we have two options, crate another copy of file for VSAN, or remove vmfs from name of the file.  As we will have the same polarion ID we can go with second option I guess.

Reply:
Removed `vmfs` from the filename.